src/frontend/context/like-context.js

```diff
@@ -0,0 +1,29 @@
+import { useReducer, createContext, useContext } from "react";
+import { LikeReducer } from "../reducer/like-reducer";
+
+const LikeContext = createContext();
+
+const LikeProvider = ({ children }) => {
+  const initialState = {
```

Comment:
You could write this `initialState` outside the provider & avoid unnecessary re-rendering of it.

Reply:
will do that